test/sql/include/plan_expand_hypertable_load.sql

```diff
@@ -68,6 +68,12 @@ INSERT INTO metrics_timestamptz SELECT generate_series('2000-01-01'::timestamptz
 INSERT INTO metrics_timestamptz SELECT generate_series('2000-01-01'::timestamptz,'2000-02-01'::timestamptz,'1d'::interval), 2;
 INSERT INTO metrics_timestamptz SELECT generate_series('2000-01-01'::timestamptz,'2000-02-01'::timestamptz,'1d'::interval), 3;
```

Comment:
```suggestion
INSERT INTO metrics_timestamptz values( '2000-03-01'::timestamptz, 3);
```

Reply:
this might change the plan output for other plans as well. But the other one should be easy to add without too much diff noise.